sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -161,6 +161,9 @@ def session_id(self):
     @session_id.setter
     def session_id(self, value):
         # type: (str) -> None
+        if value and len(value) > 128:
```

Comment:
Nit: const 128? (MAX_ID_LENGTH or something so you can use one for all?)

if you think this is overkill ignore me, this is VERY bikesheddy.

Reply:
++, updated